### PR TITLE
[7.x] Unmute more memory-related tests after the fix in #68542

### PR DESCRIPTION
**Pull request by danhermann**
Relates to #68542

### CODE REVIEW

**Comment by droberts195** [comment]
I think there was a problem with the workaround, so these tests will probably fail if unmuted. The Java security manager blocks reading `/proc/meminfo` - see for example https://gradle-enterprise.elastic.co/s/6qtcnglfzfdvg/tests/:server:test/org.elasticsearch.monitor.os.OsProbeTests/testGetTotalMemoryOnDebian8#1

**Comment by danhermann** [comment]
> I think there was a problem with the workaround, so these tests will probably fail if unmuted. The Java security manager blocks reading `/proc/meminfo` - see for example https://gradle-enterprise.elastic.co/s/6qtcnglfzfdvg/tests/:server:test/org.elasticsearch.monitor.os.OsProbeTests/testGetTotalMemoryOnDebian8#1

Ah, thanks for pointing that out, @droberts195.

**Comment by danhermann** [comment]
@droberts195, I opened https://github.com/elastic/elasticsearch/pull/68742 to address the security manager issue.

**Comment by danhermann** [comment]
#68742 has been merged and the previously failing test on the debian8 platform passed, so I think this PR is clear to be reviewed.

File: server/src/test/java/org/elasticsearch/monitor/os/OsProbeTests.java [hunk 1]
```diff
@@ -8,15 +8,8 @@
 
 package org.elasticsearch.monitor.os;
 
-import static org.hamcrest.Matchers.allOf;
-import static org.hamcrest.Matchers.anyOf;
```
Comment (probakowski) [inline server/src/test/java/org/elasticsearch/monitor/os/OsProbeTests.java:12]: These import changes are not needed for this change,  right?

**Review by probakowski — APPROVED** [review]
LGTM, one minor nit

File: server/src/test/java/org/elasticsearch/monitor/os/OsProbeTests.java (hunk 1 above)
Comment (danhermann) [reply server/src/test/java/org/elasticsearch/monitor/os/OsProbeTests.java:12]: The commit that muted these tests (https://github.com/elastic/elasticsearch/commit/09f823b1dbd64cb9b94acda7f61005a59e7aa897) moved the import statements out of the standard ordering that we use, so I just restored the original standard ordering as part of the un-mute.

**Pull request merged**